software/playground.py

```diff
@@ -11,4 +11,39 @@
 
 bb.measure()
 print(bb.cellVolts)
-print(bb.cellTemps)
\ No newline at end of file
+print(bb.cellTemps)
+
+#blame Lev below this line
+from datetime import datetime
+import csv
+
+#function takes in timestamp and voltage, writes values to csv file, "a" appends whereas "w" clears file first
+def write_to_csv(date, voltage):
+    with open("timestamps.csv", "a", newline = "") as csv_file:
+        writer = csv.writer(csv_file)
+        writer.writerow([date, voltage])
+        #writer.writerow("\n")
+
+#function clears csv file, just scaffolding for now, "w+" truncates the file
+def clear_csv(): 
+    with open("timestamps.csv", "w+", newline = "") as csv_file:
+        pass
+
+#function makes a timestamp
+def stamper():
+    dt = datetime.now()
+    iso_date = dt.isoformat()
+    return iso_date
+
+check = input("Do you want to clear the csv file, good sir? \nY/n: ")
+if (check.upper == 'Y'):
```

Comment:
In Python parentheses are not need in if statements

Reply:
I like the parentheses for readability's sake - if it is bad to have them though, I will remove them. All other requested changes have been implemented and committed.

Reply:
Up to you, it is a PEP standard to not include them when not needed. PEP standards are more like guidelines than rules.